realsense_camera/launch/includes/nodelet.launch.xml

```diff
@@ -22,18 +22,9 @@
     <param name="ir_frame_id"             type="str"  value="$(arg camera)_ir_frame" />
     <param name="ir2_frame_id"            type="str"  value="$(arg camera)_ir2_frame" />
 
-    <remap from="camera/depth/image_raw"    to="$(arg camera)/$(arg depth)/image_raw" />
-    <remap from="camera/color/image_raw"    to="$(arg camera)/$(arg rgb)/image_raw" />
-    <remap from="camera/ir/image_raw"       to="$(arg camera)/$(arg ir)/image_raw" />
-    <remap from="camera/ir2/image_raw"      to="$(arg camera)/$(arg ir2)/image_raw" />
-    <remap from="camera/depth/points"       to="$(arg camera)/$(arg depth)/points" />
-    <remap from="camera/depth/camera_info"  to="$(arg camera)/$(arg depth)/camera_info" />
-    <remap from="camera/color/camera_info"  to="$(arg camera)/$(arg rgb)/camera_info" />
-    <remap from="camera/ir/camera_info"     to="$(arg camera)/$(arg ir)/camera_info" />
-    <remap from="camera/ir2/camera_info"    to="$(arg camera)/$(arg ir2)/camera_info" />
-    <remap from="camera/get_settings"       to="$(arg camera)/get_settings" />
-    <remap from="camera/set_power"          to="$(arg camera)/set_power" />
-    <remap from="camera/force_power"        to="$(arg camera)/force_power" />
-    <remap from="camera/is_powered"         to="$(arg camera)/is_powered" />
+    <remap from="depth"    to="$(arg depth)" />
```

Comment:
@kevincwells Does this actually give the user an option to rename the namespaces? Instead, shouldnt they be set as parameters in this file and the code updated to read those parameters and create the node handles accordingly.
E.g  
- Add **param name="depth"             type="str"  value="$(arg depth)"**
- In "void BaseNodelet::getParameters()", add **pnh_.param("depth", depth_, DEPTH_NAMESPACE);** where depth_ is a global string variable.
- In "void BaseNodelet::advertiseTopics()", add **ros::NodeHandle depth_nh(nh_, depth_);** instead of ros::NodeHandle depth_nh(nh_, DEPTH_NAMESPACE);